content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md

```diff
@@ -0,0 +1,12 @@
+---
+date: '2022-11-09T11:00:00.000Z'
+externalUrl: 'https://www.edx.org/course/introduction-to-cilium'
+ogImage: cover.jpg
+categories:
+  - Getting Started
+place: Online
+title: 'Introduction to Cilium'
+ogSummary: 'Get a practical introduction to using Cilium as the networking plug-in for Kubernetes, including installation, observability with Hubble, securing network connections, and multi-cluster support - all based on eBPF for scalability, performance, and security.'
+draft: false
+from: 'edX'
```

Comment:
```suggestion
from: 'Linux Foundation'
```

Reply:
Done! Please check

Reply:
But it was necessary to hide the 'Online' tag to fit on a single line.